qemu/tests/cfg/openflow_acl_test.cfg

```diff
@@ -76,7 +76,7 @@
             ftp:
                 setup_targets = localhost
             website:
-                access_cmd_linux = lynx --dump www.redhat.com
+                access_cmd_linux = curl www.redhat.com
```

Comment:
@xiangchunfu Thanks for review. Make sense. Already update the code. Please help to review again. Thanks.